pkg/visor/init.go

```diff
@@ -344,6 +345,59 @@ func initDmsgCtrl(ctx context.Context, v *Visor, _ *logging.Logger) error {
 	return nil
 }
 
+func initDmsgHTTPLogServer(ctx context.Context, v *Visor, log *logging.Logger) error {
+	dmsgC := v.dmsgC
+	if dmsgC == nil {
+		return nil
```

Comment:
Should we not return an error here? We basically cannot initialize the log server here at all...

Reply:
Will return an error